### DISCUSSION TITLE
Confluence Integration with Mermaid

### DISCUSSION THREAD

**Discussion by davnnis2003** [mermaid-js/mermaid#3992] [General]
Hey everyone,

I have been a fan of Mermaid for a while and always look for opportunities to use it. It is very nice to see that both GitHub and also GitLab supports the syntax of Mermaid in Markdown files (even Notion supports this!)

Like other companies, the company I am working in is also using Confluence as the primary tech documentation solution. Yet, it doesn't seem Confluence is supporting Mermaid natively yet (not without paying for an extension via Marketplace). So I wonder if there is any integration plan at Mermaid side or Atlassian to provide native support in Confluence (i.e. so I could use Mermaid to build diagrams for free? xD)

This is the first time I launch a discussion in open source projects in GitHub so please let me know if there is something I missed. Happy to correct it.

**Comment by andorsk** [comment]
I would love this feature as well. To date, I've had to use an extension:

1) it makes me slightly uncomfortable as I prefer not to use extensions if possible
2) the extension isn't very good tbh. 

Personally, I would want a Github like entry point, where I could natively write mermaid in some codeblock ```mermaid``` and it would render the appropriate block.

**Comment by zhifeiyue** [comment]
We made a new Mermaid app for Confluence last week, try it:  [Mermaid Live Editor for Confluence](https://marketplace.atlassian.com/apps/1231571/mermaid-live-editor-for-confluence?hosting=cloud&tab=reviews)

**Reply by perzeababa** [reply]
Is there a reason why the released plugin for confluence and jira are only for cloud users? Can a datacenter compatible plugin be available as well?
